CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+### Setup
+
+We recommend that you use [development containers](https://containers.dev/) to contribute to this integration.
+
```

Comment:
How about adding:

If you are new to dev containers, check out a tutorial for your IDE, such as [this one for VS Code](https://containers.dev/).